core/src/main/java/io/kestra/core/models/tasks/NamespaceFiles.java

```diff
@@ -0,0 +1,23 @@
+package io.kestra.core.models.tasks;
+
+import io.micronaut.core.annotation.Introspected;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import javax.validation.Valid;
+
+@Builder
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Introspected
+public class NamespaceFiles {
+    @Valid
+    private List<String> includes;
```

Comment:
```suggestion
    private List<String> include;
```
given that it's a list, it seems better to use include as in "Include all those listed paths"